### PR TITLE
ci: add lint config

### PR DESCRIPTION
**Pull request by daixiang0**
Sync from `kubeedge/kubeedge`.

Signed-off-by: Long <long0dai@foxmail.com>

### CODE REVIEW

File: agent/pkg/tunnel/protocol/tcp/tcpproxy.go [hunk 1]
```diff
@@ -121,7 +120,7 @@ func (tp *TCPProxyService) GetProxyStream(targetNodeName, targetIP string, targe
 		if err != nil {
 			return nil, fmt.Errorf("connect to %s err: %v", targetNodeName, err)
 		}
-		klog.V(4).Infof("Data transfer between %s is realy mode", targetNodeName)
+		klog.V(4).Infof("Data transfer between %s is really mode", targetNodeName)
```
Comment (ZBoIsHere) [inline agent/pkg/tunnel/protocol/tcp/tcpproxy.go:123]: data transfer mode: relay or p2p

Comment (daixiang0) [reply agent/pkg/tunnel/protocol/tcp/tcpproxy.go]: Done.

**Pull request merged**